CMakeLists.txt

```diff
@@ -91,6 +94,35 @@ if(USE_CUDA)
   add_compile_definitions(__CUDA)
 endif()
 
+if(USE_ROCM)
+  set(CMAKE_CXX_STANDARD 11) 
+  SET(CMAKE_HIP_STANDARD 11)
+  # set(CMAKE_CXX_STANDARD_REQUIRED True) 
+  list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)
+  list(APPEND CMAKE_SYSTEM_PREFIX_PATH "/opt/rocm/hip")
+  include_directories(
+    /opt/rocm/hip/include
```

Comment:
These include directories should be found after `find_package` function.

Reply:
i. Is it OK to leave out the `include_directories` lines? From my knowledge, `target_link_libraries hip_<something>` also performs such operation on CMake 3.18+ .
ii. Is there any generated variable such as `${HIP_INCLUDE_DIR}` or something just like that(I'm not sure about the name) for you to use in `include_directories`?